.bcr/metadata.template.json

```
@@ -0,0 +1,13 @@
+{
+  "homepage": "https://github.com/grpc-ecosystem/grpc-gateway",
```

Should this link to the docs site?
```suggestion
  "homepage": "https://grpc-ecosystem.github.io/grpc-gateway",
```